Added a loading screen

Himanshu-370 opened the pull request.

shreya024 commented:
@Himanshu-370 Add Screenshots and short video to help us know what this change in the PR is all about and state the related issue

Himanshu-370 commented:
First I installed a new package named react-Lottie.
And added it to the main App.js file with a condition to show a loading screen whenever the website is refreshed.
![Screenshot (108)](https://user-images.githubusercontent.com/96609252/183002109-93c63565-1919-4a30-aa20-1bb40d7723eb.png)

Added a new file in the components folder named Preloader.jsx which has all the code related to the loading screen.
![Screenshot (109)](https://user-images.githubusercontent.com/96609252/183002120-3a94af1c-7f52-4947-a2fa-82ac31042a8d.png)

Added a new CSS file in the components folder named Preloader.css.
![Screenshot (110)](https://user-images.githubusercontent.com/96609252/183002122-defa1124-eeb6-499c-8e1c-f9bd94a42da9.png)

Final output:
![Screenshot (111)](https://user-images.githubusercontent.com/96609252/183002124-9e200324-1fbc-4f7d-aa96-c4574bd90069.png)

shreya024 commented:
> @shreya024
> 
> First I installed a new package named react-Lottie. And added it to the main App.js file with a condition to show a loading screen whenever the website is refreshed. ![Screenshot (108)](https://user-images.githubusercontent.com/96609252/183002109-93c63565-1919-4a30-aa20-1bb40d7723eb.png)
> 
> Added a new file in the components folder named Preloader.jsx which has all the code related to the loading screen. ![Screenshot (109)](https://user-images.githubusercontent.com/96609252/183002120-3a94af1c-7f52-4947-a2fa-82ac31042a8d.png)
> 
> Added a new CSS file in the components folder named Preloader.css. ![Screenshot (110)](https://user-images.githubusercontent.com/96609252/183002122-defa1124-eeb6-499c-8e1c-f9bd94a42da9.png)
> 
> Final output: ![Screenshot (111)](https://user-images.githubusercontent.com/96609252/183002124-9e200324-1fbc-4f7d-aa96-c4574bd90069.png)

Merged  🎉 Great work @Himanshu-370
